modules/ROOT/partials/configuration/advtemplate_list.adoc

```diff
@@ -0,0 +1,44 @@
+[[advtemplate_list]]
+== `+advtemplate_list+`
+
+This options allows for the loading of templates from external data sources. 
+
+To load templates from an external source, configure the `advtemplate_list` option to return a `Promise` containing the initial template list.
+
+*Type:* `+Object+`
+
+=== Example: using `+advtemplate_list+` to load a template collection
+
+[source,js]
+----
+tinymce.init({
+  plugins: 'advtemplate',
+  toolbar: 'advtemplate',
+  advtemplate_list: () => {
+      return Promise.resolve([
+        {
+          id: 1,
+          title: 'Template 1'
+        },
+        {
+          id: 2,
+          title: 'Category 2',
+          items: [
+            {
+              id: 3,
+              title: 'Template 2.1',
+            },
+            {
+              id: 4,
+              title: 'Category 2.2',
+              items: []
+            }
+          ]
+        }
+    ]);
+  }
+})
+----
+
+NOTE: A unique id value must be provided for every input item.
```

Comment:
```suggestion
[[advtemplate_templates]]
== `+ advtemplate_templates +`

This options allows for the loading of templates from external data sources. 

To load templates from an external source, configure the `advtemplate_templates ` option to return a `Promise` containing the initial template list.

*Type:* `+Object+`

=== Example: using `+ advtemplate_templates +` to load a template collection

[source,js]
----
tinymce.init({
  plugins: 'advtemplate',
  toolbar: 'addtemplate inserttemplate',
  advtemplate_templates: () => {
      return Promise.resolve([
        {
          id: 1,
          title: 'Template 1'
        },
        {
          id: 2,
          title: 'Category 2',
          items: [
            {
              id: 3,
              title: 'Template 2.1',
            },
            {
              id: 4,
              title: 'Category 2.2',
              items: []
            }
          ]
        }
    ]);
  }
})
----

NOTE: A unique id value must be provided for every input item.
```
This option is called `advtemplate_templates`

Reply:
@MitchCTiny Actually, your suggestion is not correct. 
The `advtemplate_templates` option should take a predefined list of templates (not a function), similar to the `templates` option in the old template plugin. The other options are meant for interacting with the external service. The `advtemplate_templates` option is only used to demonstrate the plugin's features with minimal configuration. Once the user sets it, the rest of the options are set automatically and communicate with in-memory storage. Therefore, after reloading the page, the template list is reset to the initial predefined list.

Reply:
Right, I'll update the suggestion.